### DISCUSSION TITLE
are the scripts for the arXiv article available?

### DISCUSSION THREAD

**Discussion by KarimElgammal** [CederGroupHub/chgnet#66] [Q&A] [answered]
Hi!

I was wondering if the scripts that calculated figure 4e and figure 6 are available for calculation on similar systems?

thanks in advance!

**Comment by bowen-bd** [comment, marked as answer]
Hi,

The scripts for figure 4e is basically from reading the MAGMOM from the MD trajectory.
You can read the ASE trajectory documentation.

Figure 6 is a diffusivity calculation that you can get from ref 63:
https://www.nature.com/articles/s41524-018-0074-y

**Reply by KarimElgammal** [reply]
thanks allot for your kind reply @BowenD-UCB 
(maybe a basic question, im novice with ML potentials) in Fig 6: I was wondering what should my script using CHGNet to calculate the Li diffusivity implement? what should be the algorithm giving the diffusivity looks like?

I like the results and I want to reproduce the nice Arrhenius plot using CHGNet of Li diffusivity in LMO for example, it will be much faster than the AIMD for sure
